.github/workflows/Deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy
+
+on:
+  push:
+    branches: 
+      - master
+      # Deploy on hack branches (keeps env clean and prevents data loss on deploy)
+      - hack/* 
```

Comment:
Why `hack/*` branches? I don't fully understand the comment

Reply:
Will that deploy them to a separate app? Or will that override the prod one?

Reply:
If we use `master` for our main app, we'll have one app environment and one DB. I guess we can just delete the app afterwards, but my thought was we'd have a new instance of the app for each hack. Want to just stick with `master`?

Reply:
Ohhh gotcha. No, I like that. That makes it easy to clean it up too, because we won't need to worry about old Slack spaces pointing to an app for another hackathon. I assume they would all have different urls?

Reply:
That's the plan but I don't want to have to figure out how to derive the app name from the branch name just yet since the only available one has a full ref path associated with it.